tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,415 @@
+
+		$post_id = Tribe__Main::post_id_helper();
+
+		// Awaiting https://central.tri.be/issues/80685 instead of the empty() check here
+		if ( '' === $current_position && empty( $post_id ) ) {
+			$current_position = $default_position;
+		}
+
+		$output = sprintf( '%1$s<option value=""></option>%1$s', PHP_EOL );
+
+		foreach ( $symbol_position_list_array as $key => $value ) {
+			$selected = selected( $key, $current_position, false );
+
+			$opt_text = esc_attr_x( $value, 'Currency symbol position', 'tribe-extension' );
+
+			$output .= sprintf( '<option value="%s" %s>%s</option>%s', esc_attr( $key ), esc_html( $selected ), $opt_text, PHP_EOL );
+		}
+
+		return $output;
+	}
+
+	/**
+	 * Detect which mode of the Community Events form we are currently in.
+	 *
+	 * @param null $page
+	 *
+	 * @return bool
+	 */
+	protected function is_community_page( $page = null ) {
+		$main = tribe( 'community.main' );
+
+		$event_post_type = Tribe__Events__Main::POSTTYPE;
+		$action          = Tribe__Utils__Array::get( $main->context, 'action' );
+		$ce_post_type    = Tribe__Utils__Array::get( $main->context, 'post_type', $event_post_type ); // assume event post type if not set
+
+		switch ( $page ) {
+			case 'add-event':
+				if (
+					'add' === $action
+					&& $event_post_type === $ce_post_type
+				) {
+					$is_page = true;
+				}
+				break;
+
+			case 'edit-event':
+				if (
+					'edit' === $action
+					&& $event_post_type === $ce_post_type
+				) {
+					$is_page = true;
+				}
+				break;
+
+			default:
+				$is_page = false;
+				break;
+		}
+
+		return $is_page;
+	}
+
+	/**
+	 * Do the output
+	 *
+	 * @return string
+	 */
+	public function display() {
+		$symbol_list = $this->symbol_list_select_options();
+
+		// if no symbols are entered in Settings, do not display Symbol or Position on CE form
+		if ( empty( $symbol_list ) ) {
+			return '';
+		}
+
+		?>
+
+		<table class="tribe-section-content">
+			<colgroup>
+				<col class="tribe-colgroup tribe-colgroup-label">
+				<col class="tribe-colgroup tribe-colgroup-field">
+			</colgroup>
+
+			<tr class="tribe-section-content-row">
+				<td class="tribe-section-content-label">
+					<?php tribe_community_events_field_label( 'EventCurrencySymbol', esc_html__( 'Currency Symbol:', 'tribe-extension' ) ); ?>
+				</td>
+				<td class="tribe-section-content-field">
+					<select
+						id="EventCurrencySymbol"
+						aria-label="<?php esc_attr_e( 'Currency Symbol', 'tribe-extension' ); ?>"
+						name="EventCurrencySymbol"
+						class="event-cost-currency-symbol tribe-dropdown"
+						placeholder="<?php esc_attr_e( 'Select a currency symbol', 'tribe-extension' ); ?>"
+					>
+						<?php echo $symbol_list; ?>
+					</select>
+				</td>
+			</tr>
+
+		</table>
+
+		<?php
+		// allow disabling the Before/After choice on the CE form
+		$disabled = tribe_get_option( 'tribe_ext_ce_cost_currency_symbol_position_disabled' );
+
+		if ( empty( $disabled ) ) {
```

Comment:
Make it a one-liner.

Reply:
Like this?
`if ( empty( tribe_get_option( 'tribe_ext_ce_cost_currency_symbol_position_disabled' ) ) ) {`

If yes, that's not compatible with PHP versions below 5.5

Reply:
One liner like `if ( conditional ) :`

Reply:
got it. done!